app/lib/shared/urls.dart

```diff
@@ -98,17 +99,18 @@ void syntaxCheckHomepageUrl(String url) {
   try {
     uri = Uri.parse(url);
   } catch (_) {
-    throw new Exception('Unable to parse homepage URL: $url');
+    throw new GenericProcessingException('Unable to parse homepage URL: $url');
   }
   if (!uri.hasScheme || !uri.scheme.startsWith('http')) {
```

Comment:
This should be a stronger check as it might allow httpfoo://, strenghen it to check for "http://" and "https://". (Doesn't need to happen in this CL)

Reply:
I'll do it in a subsequent PR, as it will need to have tests too.